pkg/lib/rolesgroups/store_group_user.go

```diff
@@ -37,7 +38,8 @@ func (s *Store) ListGroupsByUserID(userID string) ([]*Group, error) {
 	).
 		From(s.SQLBuilder.TableName("_auth_user_group"), "ug").
 		Join(s.SQLBuilder.TableName("_auth_group"), "g", "ug.group_id = g.id").
-		Where("ug.user_id = ?", userID)
+		Where("ug.user_id = ?", userID).
```

Comment:
Not related to this issue, but it seems this `ListGroupsByUserID` can simply call `ListGroupsByUserIDs` with a single user id instead of duplicating the implementation.

Reply:
Actually yes, any concern for that? @louischan-oursky

Reply:
No concern.